internal/exec/util/file.go

```diff
@@ -112,6 +114,33 @@ func RenderFile(l *log.Logger, c *resource.HttpClient, f types.File) *File {
 		return nil
 	}
 
+	if f.User.Name != "" {
+		u, err := user.Lookup(f.User.Name)
```

Comment:
I tested this by having it lookup the `core` user and it worked, I didn't attempt to add a user ignition was also creating due to the `sysusers` issue.

The cgo function isn't exported, think I should copy it into ignition?